packages/core/src/__tests__/feature-app-manager.test.ts

```diff
@@ -185,79 +175,107 @@ describe('FeatureAppManager', () => {
       ]);
     });
 
-    describe('with a Feature App definition that is failing the externals validation', () => {
-      let mockError: Error;
-
+    describe('with an ExternalsValidator provided to the FeatureAppManager', () => {
```

Comment:
Don't we need another test that tests the declared externals to be ignored if no validator is configured?

Reply:
yes

Reply:
done